app/models/user-store/user-store.ts

```diff
@@ -289,6 +301,18 @@ export const UserStoreModel = types
         superLikeStatusResult
       ]: [UserResult, SuperLikeStatusResult] = yield userFetchPromise
 
+      if (Platform.OS === "ios") {
+        // Saved auth cookie for share extension
+        try {
+          const { likecoin_auth: authCookie }: Cookies = yield CookieManager.get(self.env.likeCoAPI.apisauce.getBaseURL())
+          if (authCookie) {
+            yield EncryptedStorage.setItem(authCookie.name , JSON.stringify(authCookie))
```

Comment:
Since we use hardcode key to access and remove, we should use hardcode key to set?
```suggestion
            yield EncryptedStorage.setItem('likecoin_auth' , JSON.stringify(authCookie))
```

Reply:
hardcoded `likecoin_auth` in line 307

Reply:
Kind of confusing but anyway